docs/api/12_customer_usage/customer-usage-object.mdx

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# The customer usage object
+
+This object represents the charge usage of a customer of your business on the current billing period.<br/>
```

Comment:
"This object represents the usage-based charges associated with one of your customers for the current billing period."